src/pages/_app.tsx

```diff
@@ -19,10 +21,15 @@ import 'styles/global.scss'
 
 import { Provider } from 'mobx-react'
 import { stores } from 'stores/DomainStores/stores'
-
+const {
+  publicRuntimeConfig: { STRIPE_PUBLIC_KEY },
+} = getConfig()
 // Client-side cache, shared for the whole session of the user in the browser.
 const clientSideEmotionCache = createEmotionCache()
 
+// Stripe promise needed for stripe elements
+export const stripePromise = loadStripe(STRIPE_PUBLIC_KEY)
```

Comment:
I wonder if this is the best place for the stripe promise and stripe loader as it will be loaded on pages without any payment needs.

@dimitur2204  Do you think we can define it in a helper and include that helper wherever we need it?

Reply:
Take a look at what I've done here. What exactly do you have in mind by a helper?
https://github.com/podkrepi-bg/frontend/pull/1287/commits/b13b5b144bff094694b5c3a6ff485335cb8e97d0

Reply:
Looks great! Thanks for adding that :+1: